alica_ros_turtlesim/alica/etc/Alica.conf

```diff
@@ -0,0 +1,60 @@
+[Alica]
+	#ms
+	TeamTimeOut = 30000
+	AssignmentProtectionTime = 500
+
+	#Hz
+	MinBroadcastFrequency = 15
+	MaxBroadcastFrequency = 15
+	EngineFrequency = 30 # per plan, per iteration:
+
+	MaxRuleApplications = 20
+	
+	#if true, do not send out messages (except debug status) till MaySendMessages is set to true
+	SilentStart = false
+	AllowIdling = false
+	UseStaticRoles = true
+
+	#relative to ES_ROOT env variable or absolute:
+	PlanDir = plans
+	RoleDir = roles
+	MiscDir = Misc
+
+	[TeamBlackList]
+		InitiallyFull = false
+	[!TeamBlackList]
+
+	[StatusMessages]
+		Enabled = true
+		Frequency = 10
+	[!StatusMessages]
+
+	[EventLogging]
+		Enabled = true
+		LogFolder = log/temp
+	[!EventLogging]
+
+	[CycleDetection]
+		Enabled = true
+		CycleCount = 5
+		MinimalAuthorityTimeInterval = 800
+		MaximalAuthorityTimeInterval = 5000
+		IntervalIncreaseFactor = 1.5
+		IntervalDecreaseFactor = 0.999 #about 1min cool down
+		MessageTimeInterval = 60
+		MessageWaitTimeInterval = 200
+		HistorySize = 45
+	[!CycleDetection]
+
+	[CSPSolving]
+		EnableCommunication = true
+		CommunicationFrequency = 10
+		SeedMergingThreshold = 0.0015
+		ConflictThreshold = 1000
+		SeedTTL4Communication = 250
+		SeedTTL4Usage = 5000
+		MaxSolveTime = 1000
```

Comment:
Reduce the maxSolveTime to be more inline with the behavior frequency.
Typical values should be below 30ms, e.g, 10ms